posthog/clickhouse/log_entries.py

```diff
@@ -0,0 +1,68 @@
+from posthog.clickhouse.kafka_engine import KAFKA_COLUMNS, kafka_engine, ttl_period
+from posthog.clickhouse.table_engines import ReplacingMergeTree
+from posthog.kafka_client.topics import KAFKA_LOG_ENTRIES
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+LOG_ENTRIES_TABLE = "log_entries"
+LOG_ENTRIES_TTL_WEEKS = 1
+
+LOG_ENTRIES_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER '{cluster}'
+(
+    team_id UInt64,
+    log_source LowCardinality(String),
+    log_source_id VARCHAR,
+    instance_id VARCHAR,
```

Comment:
What's the difference between `log_source_id` and `instance_id` here?

I guess for batch exports you have the export and a specific run of it?

I don't know we'd always want two ids... although willing to be convinced.

Would it be better to have a single `log_id` and for batch exports you could concatenate the two ids? Or not have the instance id in the sort key?

batch exports would then query `log_id = f"{first_id}/{second_id}"` or `log_id like f"{first_id}/%'... but you know your query pattern and I don't so  this is very much me thinking out loud. 

----

I could always write console logs with two ids by including session id and pageview id for example.

----

So, at least we should have a SQL comment in the table definition explaining the intended use of the two fields for the future traveller.

Reply:
It's better to have an extra id field and you to not use it than do concatenations at query time